src/Goerli-DssSpellCollateral.sol

```diff
@@ -28,61 +61,257 @@ contract DssSpellCollateralAction {
     // $ bc -l <<< 'scale=27; e( l(1.08)/(60 * 60 * 24 * 365) )'
     //
     // A table of rates can be found at
-    //    https://ipfs.io/ipfs/QmTRiQ3GqjCiRhh1ojzKzgScmSsiwQPLyjhgYSxZASQekj
+    //    https://ipfs.io/ipfs/QmX2QMoM1SZq2XMoTbMak8pZP86Y2icpgPAKDjQg4r4YHn
```

Comment:
IPFS giving me trouble, but I have this as the most recent rates IPFS file: `QmPgPVrVxDCGyNR5rGp9JC5AUxppLzUAqvncRJDcxQnX1u`

We should confirm.  IIRC, the new file is a 5bps step.  I can't load either file at the moment.

Reply:
Did you manage to confirm that's the right hash?

Reply:
@godsflaw Looks like `QmX2QMoM1SZq2XMoTbMak8pZP86Y2icpgPAKDjQg4r4YHn` is the latest one:

https://www.diffchecker.com/I6Ykw2ys